Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/ControllerPoseSynchronizer.cs

```diff
@@ -51,17 +46,8 @@ public bool DestroyOnSourceLost
         /// <inheritdoc />
         public virtual IMixedRealityController Controller
         {
-            get { return controller; }
-            set
-            {
-                controller = value;
-
-                if (controller != null && gameObject != null)
-                {
-                    handedness = value.ControllerHandedness;
-                    gameObject.name = $"{handedness}_{gameObject.name}";
```

Comment:
Why remove the setting of the game object name, unless you are now doing this somewhere else?

Reply:
In the PR description there is an example. Basically constant calling to his makes "Left_Right_Left_BlahPointer" names. Instead of checking the name and doing substrings, I just removed it so the name is left as-is